Makefile

```diff
@@ -1,5 +1,5 @@
 setup:
-	rm -rf node_modules; npm install
+	rm -rf node_modules; rm package-lock.json; npm install
```

Comment:
Why we are removing lock?
Where is a guy who recomends not to do that
https://betterprogramming.pub/npm-ci-vs-npm-install-which-should-you-use-in-your-node-js-projects-51e07cb71e26?gi=2545aa193f4

Reply:
okay, done